pkg/controller/elasticsearch/client/v6.go

```diff
@@ -148,6 +149,22 @@ func (c *clientV6) DeleteVotingConfigExclusions(ctx context.Context, waitForRemo
 	return errors.New("Not supported in Elasticsearch 6.x")
 }
 
+func (c *clientV6) DeleteAutoscalingPolicies(_ context.Context) error {
+	return errors.New("Not supported in Elasticsearch 6.x")
+}
+
+func (c *clientV6) CreateAutoscalingPolicy(_ context.Context, _ string, _ esv1.AutoscalingPolicy) error {
+	return errors.New("Not supported in Elasticsearch 6.x")
```

Comment:
Nit: only because my linter complains; maybe we can define this error once? I don't feel strongly about it though.